client/blocks/author-compact-profile/index.jsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Gravatar from 'components/gravatar';
+import ReaderAuthorLink from 'components/reader-author-link';
+import ReaderSiteStreamLink from 'components/reader-site-stream-link';
+import ReaderFollowButton from 'reader/follow-button';
+import { localize } from 'i18n-calypso';
+
+const AuthorCompactProfile = React.createClass( {
```

Comment:
hrm, re-read the `blocks` readme and we're probably fine in `blocks`. Though I'm not sure where the line is between primitive and complex entity...